src/games/strategy/triplea/ui/ProductionPanel.java

```diff
@@ -121,7 +121,8 @@ protected ProductionPanel(final IUIContext uiContext) {
   }
 
   // made this protected so can be extended by edit production panel
-  protected void initRules(final PlayerID player, final GameData data,
+  protected void initRules(final PlayerID player,
+      @SuppressWarnings("unused") final GameData data,
```

Comment:
There is a child class that overrides the method, it needs the parameter. The suppresswarnings is not a clean fix, but it is quick to get rid of the warning .